src/D2L.Bmx/Aws/AwsCredsCache.cs

```diff
@@ -0,0 +1,64 @@
+using System.Runtime.InteropServices;
+using IniParser;
+using IniParser.Model;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+namespace D2L.Bmx.Aws;
+
+internal class AwsCredsCache() {
+	private static readonly JsonSerializerOptions _options =
+		new() { DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull };
+	public void SaveToFile( string Org, string User, AwsRole role, AwsCredentials credentials ) {
+		if( !Directory.Exists( BmxPaths.BMX_DIR ) ) {
+			return;
+		}
+		if( Org == "" || User == "" ) { //It's not set in config file
+			return;
+		}
```

Comment:
What's this for? To not save if we don't have a config file?

Reply:
I want both Org and User already set in the config file. So I can label each Aws session token as which user who it belongs to.